test/plugin_functional/test_suites/core_plugins/application_deep_links.ts

```diff
@@ -39,8 +39,7 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
     });
   };
 
-  // Failing: See https://github.com/elastic/kibana/issues/166893
-  describe.skip('application deep links navigation', function describeDeepLinksTests() {
+  describe('application deep links navigation', function describeDeepLinksTests() {
```

Comment:
I can't reproduce a failed test locally and CI seems happy as it is.
Opting for the unskip first and see how long it holds 🤞

Reply:
Perhaps the CI pass was just luck.
The[ flaky test runner](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/3494)  failed these tests.

For example, the test for deep links fails with:

```
fail: core plugins application deep links navigation should navigate to Home when navlink is clicked inside the defined category group
[00:06:37]         │      Error: timed out waiting for Url to be http://localhost:5620/app/dl/home
[00:06:37]         │       at onFailure (retry_for_truthy.ts:39:13)
[00:06:37]         │       at retryForSuccess (retry_for_success.ts:59:13)
[00:06:37]         │       at retryForTruthy (retry_for_truthy.ts:27:3)
[00:06:37]         │       at RetryService.waitFor (retry.ts:59:5)
[00:06:37]         │       at Context.<anonymous> (application_deep_links.ts:75:7)
[00:06:37]         │       at Object.apply (wrap_function.js:73:16)
```

Reply:
@elastic/appex-sharedux I could use a hand in figuring out why tests using `appMenu.clickLink` fail in CI and pass locally. The most recent changes to relevant components seems to be with `CollapsibleNav` but that was 4 months ago.